webui/react/src/routes/utils.ts

```diff
@@ -127,3 +131,7 @@ export const routeAll = (path: string): void => {
     history.push(getReactPath(path), { loginRedirect: clone(window.location) });
   }
 };
+
+export const linkPath = (path: string, external = false): string => {
```

Comment:
**suggestion:** rename to something like `addBasePath`?

Reply:
hmm I see your point currently it's doing that sometimes but not other times. 
I see us adding more logic to this, eg adding serverAddress to external paths that don't specify the host. (in #1551) so I think a name that could encompass doing different things to prepare the path would work better although naming things always have a subjective portion :)